servicex/templates/app/configmap.yaml

```diff
@@ -91,7 +94,6 @@ data:
     {{- else }}
     TRANSFORMER_X509_SECRET=None
     {{- end }}
-    TRANSFORMER_VALIDATE_DOCKER_IMAGE = {{- ternary "True" "False" .Values.app.validateTransformerImage }}
```

Comment:
Why would you delete this? - This is a setting that states whether to validate the provided transformer image exists on DockerHub. We want this in prod, but not in development